proposals/readonly-methods.md

```diff
@@ -0,0 +1,65 @@
+# Readonly Methods
+
+* [x] Proposed
+* [ ] Prototype: Not Started
+* [ ] Implementation: Not Started
+* [ ] Specification: Not Started
```

Comment:
Please move the state tracking to the issue. This is how we've been handling recent proposals.

Reply:
Fixed.